chia/util/api_decorators.py

```diff
@@ -27,7 +27,7 @@ class ApiMetadata:
     bytes_required: bool = False
     execute_task: bool = False
     reply_types: List[ProtocolMessageTypes] = field(default_factory=list)
-    message_class: Optional[Any] = None
+    message_class: Any = None
```

Comment:
I think `Optional[]` adds value here; it's a more specific type than just `Any`.
A better type would be:
```
message_class: Optional[Type[Streamable]] = None
```

Reply:
Right, `Type[Streamable]` makes sense.